src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readFile(fromFileName)));
+    }
+
+    private String readFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(",");
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        return stringBuilder.toString();
+    }
 
+    private String createReport(String data) {
+        int buy = 0;
+        int supply = 0;
+        String[] dataArray = data.split(",");
+        for (int i = 0; i < dataArray.length; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < dataArray.length; i += 2) {
```
just a suggestion, this will create less iterations